lib/fluent/plugin/out_google_cloud.rb

```diff
@@ -167,7 +167,21 @@ def write(chunk)
           'entries' => [],
         }
         arr.each do |time, record|
-          if (record.has_key?('timeNanos'))
+          if (record.has_key?('timestamp') &&
+              record['timestamp'].has_key?('seconds') &&
+              record['timestamp'].has_key?('nanos'))
```

Comment:
I debated that but decided not to - since this is an interim step I really only want people to use this if they have a need, and whole seconds can already be handled by the native fluentd "time" using time_format %s.